packages/relay/package.json

```diff
@@ -3,8 +3,15 @@
   "version": "0.1.2",
   "description": "SWC plugin for relay",
   "main": "swc_plugin_relay.wasm",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.js",
+    "default": "./dist/index.js"
+  },
   "scripts": {
-    "prepack": "cp ../../target/wasm32-wasi/release/swc_plugin_relay.wasm ."
+    "build": "tsup index.ts --format cjs,esm --dts",
```

Comment:
What's usecase for tsup in here? Specifically, what's major problem cannot be resolved with plain tsc for this relatively small package?

Reply:
It was for dual packaging, but I can change it to only use tsc and then make the package CJS only if you want.

Reply:
Before changing this, I think we need to resolve https://github.com/swc-project/plugins/pull/38#discussion_r880045731 question first: who can actually import in cjs / esm in normal node.js environment and how?

Reply:
I left a comment there :)